### DISCUSSION TITLE
why search object is not showing objects while setting the many to many object relation in pimcore 10?

### DISCUSSION THREAD

**Discussion by tkshitiz** [pimcore/pimcore#12576] [Q&A / Support]
![Screenshot from 2022-07-04 11-59-03](https://user-images.githubusercontent.com/52815756/177154699-184c3ffa-ced1-4135-bd8f-b16a3e7d0ab7.png)

**Comment by langert** [comment]
Hi,
I got the same issue. Drag & Drop to the object relation is possible, but the listing in the dialog is always empty :-(

**Comment by SamyMP** [comment]
Hi, have you tried to run the command `pimcore:search-backend-reindex` and then retry ?

**Reply by langert** [reply]
@SamyMP made my day!
When is this command triggered usually?

**Reply by SamyMP** [reply]
I'm glad it helped :)

Not sure, but this task might be part of the maintenance... If it isn't, you could try to create a cron to run this command on a regular basis (although it can be very long). Here are other commands available : https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Deployment_Tools.html#page_Pimcore-Console
